java.lang.IndexOutOfBoundsException: 0
occurred in the presentation compiler.

action parameters:
offset: 925
uri: file://<WORKSPACE>/src/test/scala/V_Single-Cycle/TAIL_MASK/TAIL_MASKTest.scala
text:
package Vpractice

import chisel3._
import org.scalatest.FreeSpec
import chiseltest._

class TAIL_MASKTest extends FreeSpec with ChiselScalatestTester{
   "TAIL_MASKTest test" in{
       test(new Tail_Mask()){c =>
         c.io.v_data1(0).poke(5.U)
         c.io.v_data1(1).poke(5.U)
         c.io.v_data1(2).poke(5.U)
         c.io.v_data1(3).poke(5.U)
         c.io.v_data1(4).poke(5.U)
         c.io.v_data1(5).poke(5.U)
         c.io.v_data1(6).poke(5.U)
         c.io.v_data1(7).poke(5.U)
         c.io.v_data1(8).poke(5.U)
         c.io.v_data1(9).poke(5.U)
         c.io.v_data1(10).poke(5.U)
         c.io.v_data1(11).poke(5.U)
         c.io.v_data1(12).poke(5.U)
         c.io.v_data1(13).poke(5.U)
         c.io.v_data1(14).poke(5.U)
         c.io.v_data1(15).poke(5.U)
        //  c.io.v_data2.poke()
         c.io.tm_vl.poke()
        //  c.io.tm_vm.poke(1.B)
         c.clock.step(10) 
         c.io.v_data1_out(@@).expect()
        //  c.io.v_data2_out.expect()
        
       }
    }
}       


error stacktrace:
scala.collection.LinearSeqOps.apply(LinearSeq.scala:131)
	scala.collection.LinearSeqOps.apply$(LinearSeq.scala:128)
	scala.collection.immutable.List.apply(List.scala:79)
	dotty.tools.dotc.util.Signatures$.countParams(Signatures.scala:501)
	dotty.tools.dotc.util.Signatures$.applyCallInfo(Signatures.scala:186)
	dotty.tools.dotc.util.Signatures$.computeSignatureHelp(Signatures.scala:94)
	dotty.tools.dotc.util.Signatures$.signatureHelp(Signatures.scala:63)
	scala.meta.internal.pc.MetalsSignatures$.signatures(MetalsSignatures.scala:17)
	scala.meta.internal.pc.SignatureHelpProvider$.signatureHelp(SignatureHelpProvider.scala:51)
	scala.meta.internal.pc.ScalaPresentationCompiler.signatureHelp$$anonfun$1(ScalaPresentationCompiler.scala:371)
